src/rules/converters/space-within-parens.ts

```diff
@@ -0,0 +1,18 @@
+import { RuleConverter } from "../converter";
+
+export const convertSpaceWithinParens: RuleConverter = tslintRule => {
+    let arg = "";
+    if (tslintRule.ruleArguments.length === 1) {
+        arg = "always";
```

Comment:
This is actually a behavior change from TSLint to ESLint. TSLint allows specifying a specific number of parenthesis, while ESLint enforces one (at least I think it's exactly one, not at least one - it's a little hard to tell from the docs). Could you please add a `notices` explaining that?